sormas-backend/src/main/java/de/symeda/sormas/backend/crypt/CmsReader.java

```diff
@@ -9,9 +10,16 @@
 import java.security.cert.X509CertSelector;
 import java.security.cert.X509Certificate;
 import java.util.Collection;
+import java.util.Collections;
 import java.util.Iterator;
 import java.util.List;
 
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.PropertyAccessor;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.symeda.sormas.api.sormastosormas.SormasToSormasException;
```

Comment:
`SormasToSormasException` and `SormasToSormasValidationException` don't seem to be used

Reply:
fixed